README.md

```diff
@@ -8,7 +8,15 @@ Clone the repository
 
 `git clone git@github.com:nimblehq/flutter_templates.git`
 
-TBD: setup.
+## Prerequisite
+- Flutter 2.2
+- Flutter version manager (recommend): [fvm](https://fvm.app/)
+
+## Getting Started
+
+- Create the project:
+
+  - `$ fvm flutter create --template=app --platforms=ios,android .`
```

Comment:
I don't think you need this, we already create this project and it is supposed to run this command beforehand, so you don't need to instruct this anymore. In the future, what will happen here is actually run custom command to create a project based on this template project.

Reply:
@sleepylee got it. I'll remove this part 🙏

Reply:
@sleepylee I removed it and added `TBD` 🙏